packages/generate/src/queries.ts

```diff
@@ -80,9 +102,7 @@ currently supported.`);
           }
         } catch (err) {
           wasError = true;
-          console.log(
-            `Error in file '${prettyPath}': ${(err as Error).toString()}`
-          );
+          collectAndPrintError(errs, prettyPath, err)
```

Comment:
Given the updates here, we'd change this to:

```suggestion
          errs.push(formatError(prettyPath, err));
```